src/zenml/utils/package_utils.py

```diff
@@ -0,0 +1,47 @@
+#  Copyright (c) ZenML GmbH 2024. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+"""Utility functions for the package."""
+import requests
+from packaging import version
+
+
+def is_latest_zenml_version() -> bool:
+    """Checks if the currently running ZenML package is on the latest version.
+
+    Returns:
+        True in case the current running zenml code is the latest available version on PYPI, otherwise False.
+
+    Raises:
+        RuntimeError: In case something goe wrong
+    """
+    from zenml import __version__
+
+    # Get the current version of the package
+    current_version = __version__
```

Comment:
this name feels like it might be updated? current_local_version or just local_version?